### PR TITLE
Task/fix navbar add average rating to index

### PR DESCRIPTION
**Pull request by jennywin**
@cchanningallen Tweaked the navbar a bit and added each product's average review rating on the index view. Still need to move things out into partials, to do later.

### CODE REVIEW

File: app/assets/stylesheets/bootstrap_overrides.scss [hunk 1]
```diff
@@ -10,28 +10,15 @@ $lightest-green: #dff0d8;
...
 body {
-    margin-top: 50px;
-    margin-bottom: 50px;
-    padding-top: 20px;
-    padding-bottom: 0px;
-    padding-right: 0px;
-    padding-left: 0px;
-    margin-right: 130px;
-    margin-left: 130px;
+    padding: 100px 100px 20px 100px;
```
Comment (cchanningallen) [inline app/assets/stylesheets/bootstrap_overrides.scss:20]: Nice!

File: app/models/product.rb [hunk 2]
```diff
@@ -18,4 +18,8 @@ class Product < ActiveRecord::Base
 
   extend FriendlyId
   friendly_id :name, use: [:slugged, :history]
+
+  def average_rating
+    self.reviews.blank? ? 0 : self.reviews.average(:rating).round(2)
+  end
```
Comment (cchanningallen) [inline app/models/product.rb:24]: much better location :)

File: app/views/reviews/index.html.erb [hunk 3]
```diff
@@ -4,7 +4,7 @@
   </div>
 
   <div class="panel-body">
-    <div class="star-rating" data-score= <%= @average_review %> ></div>
+    <div class="star-rating" data-score= <%= @product.average_rating %> ></div>
```
Comment (cchanningallen) [inline app/views/reviews/index.html.erb:7]: I'm surprised this does't need quotes around it, like `<div class="star-rating" data-score="<%= @product.average_rating %>" ></div>`. Learn something new erryday :)

**Comment by cchanningallen** [comment]
Looks good! Great refactor :)

File: app/views/reviews/index.html.erb (hunk 3 above)
Comment (jennywin) [inline app/views/reviews/index.html.erb:7]: I was surprised, too! :)

**Comment by jennywin** [comment]
Thank you!

**Pull request merged**